index.html

```diff
@@ -4,6 +4,7 @@
 <head>
 	<title>Advanced Components</title>
 	<link rel="stylesheet" href="./CSS/index.css">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/2.1.2/TweenMax.min.js"></script>
 </head>
 
 <body>
```

Comment:
- Great job on the assingment Chinedu. 
- You completed the MVP by implementing Dropdown functionality along with Tabs components. Not only that but - You added GASP animation on the menu
- Your code is clean and git workflow is fluid as usual, great job 👍